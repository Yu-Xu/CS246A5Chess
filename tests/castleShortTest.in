game human human
move g2 g4
move f7 f5
move g1 f3
move g7 g6
move f1 h3
move g8 f6
move e1 g1
