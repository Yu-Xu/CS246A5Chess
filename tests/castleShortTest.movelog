8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _P_
3 _ _ _ _ 
2 PPPPPP P
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 ppppp pp
6  _ _ _ _
5 _ _ _p_ 
4  _ _ _P_
3 _ _ _ _ 
2 PPPPPP P
1 RNBQKBNR
  abcdefgh
8 rnbqkbnr
7 ppppp pp
6  _ _ _ _
5 _ _ _p_ 
4  _ _ _P_
3 _ _ _N_ 
2 PPPPPP P
1 RNBQKB_R
  abcdefgh
8 rnbqkbnr
7 ppppp _p
6  _ _ _p_
5 _ _ _p_ 
4  _ _ _P_
3 _ _ _N_ 
2 PPPPPP P
1 RNBQKB_R
  abcdefgh
8 rnbqkbnr
7 ppppp _p
6  _ _ _p_
5 _ _ _p_ 
4  _ _ _P_
3 _ _ _N_B
2 PPPPPP P
1 RNBQK _R
  abcdefgh
8 rnbqkb r
7 ppppp _p
6  _ _ np_
5 _ _ _p_ 
4  _ _ _P_
3 _ _ _N_B
2 PPPPPP P
1 RNBQK _R
  abcdefgh
8 rnbqkb r
7 ppppp _p
6  _ _ np_
5 _ _ _p_ 
4  _ _ _P_
3 _ _ _N_B
2 PPPPPP P
1 RNBQ_RK 
  abcdefgh
