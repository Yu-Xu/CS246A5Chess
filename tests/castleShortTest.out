8 rnbqkb r
7 ppppp _p
6  _ _ np_
5 _ _ _p_ 
4  _ _ _P_
3 _ _ _N_B
2 PPPPPP P
1 RNBQ_RK 
